checkpoint.go

```diff
@@ -106,6 +107,12 @@ type Checker struct {
 	nextCheckAtLock sync.RWMutex
 }
 
+// Flag is an attribute of a product.
```

Comment:
The name and the comment were copied from @brb's branch; actually I think the comment is more wrong given all the other uses of the word "flag".